core/dns.go

```diff
@@ -154,36 +154,38 @@ func (dnsResolver *DNSResolver) LookupHost(hostname string) ([]net.IP, time.Dura
 // LookupCNAME uses a DNSSEC-enabled query to  records for domain and returns either
 // the target, "", or a if the query fails due to DNSSEC, error will be set to
 // ErrorDNSSEC.
-func (dnsResolver *DNSResolver) LookupCNAME(domain string) (string, error) {
+func (dnsResolver *DNSResolver) LookupCNAME(domain string) (string, time.Duration, error) {
```

Comment:
I thought we were removing DNSSEC, but if not, this needs to be named something that indicates that it's DNSSEC.